testing/xfpga/test_enum_c.cpp

```diff
@@ -994,6 +972,9 @@ TEST_P(enum_c_p, get_guid) {
   EXPECT_EQ(fpgaDestroyProperties(&prop), FPGA_OK);
 }
 
+
+
+
 INSTANTIATE_TEST_CASE_P(enum_c, enum_c_p, 
                         ::testing::ValuesIn(test_platform::platforms({})));
```

Comment:
mock_platorms() I think?

Reply:
I didn't change these tests so I'm not sure that they are mock only. I only added a new test that should be mock only.
This is instantiated on line 1074.